http_parser.c

```diff
@@ -1217,19 +1175,8 @@ size_t http_parser_execute (http_parser *parser,
 
         /* XXX allow spaces after digit? */
```

Comment:
I would say no. It's pretty clear in the RFC that the HTTP version in the request line should be followed by CRLF.